ncp-web/.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
can we get rid of this file? you might want to add this to your gitignore

Reply:
Ah, of course. Must have missed it.